modules/mocked_authentication/lib/mockdata/writer.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module MockedAuthentication
+  module Mockdata
+    class Writer
+      def self.save_credential(credential:, credential_type:)
+        parsed_email = parse_email(credential['email'])
```

Comment:
I'm running into an undefined method error here:
`2023-03-23 09:51:22.400663 E [34431:puma srv tp 003 sentry_logging.rb:57] Rails -- undefined method '[]' for nil:NilClass`

I believe because my credential object's keys are set to symbols and not strings:
```
[1] pry(MockedAuthentication::Mockdata::Writer)> credential
=> {:sub=>"381ce98f-d604-43c9-9512-2088d0983e2d",
 :iss=>"https://idp.int.identitysandbox.gov/",
 :email=>"vets.gov.user+228@gmail.com",
 :email_verified=>true,
 :verified_at=>1638553278,
 :ial=>"http://idmanagement.gov/ns/assurance/ial/1",
 :aal=>"urn:gov:gsa:ac:classes:sp:PasswordProtectedTransport:duo"}
 ```
 
 So should this be changed to `credential[:email]`?

Reply:
Working now!